app/controllers/lists_controller.rb

```diff
@@ -242,50 +252,60 @@ def modifications
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_list
-      @list = List.find(params[:id])
-    end
 
-    # Only allow a trusted parameter "white list" through.
-    def list_params
-      params.require(:list).permit(:name, :description, :is_ranked, :is_admin, :is_featured, :is_private, :custom_field_name, :short_description, :access)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_list
+    @list = List.find(params[:id])
```

Comment:
cool! so, i'm now leaning toward wanting to get formatted tags by calling `@list.formatted_tags(user)` (or `tags_for(user)`) from the view. but if i wanted to prepare a `@tags` instance variable (or something like that...), this would be the place to do it, yeah?